package.json

```diff
@@ -4,7 +4,7 @@
   "description": "React hooks and map context provider for Google Maps",
   "source": "src/index.ts",
   "main": "dist/index.umd.js",
-  "module": "dist/index.modern.js",
+  "module": "./dist/index.modern.mjs",
   "types": "dist/index.d.ts",
```

Comment:
Is the `./` at the beginning needed? If so, we probably have to do the same for lines 5, 6 and 8.

Reply:
hm no, seems not be needed 👍